tests/unit/test_hnsw_tiered.cpp

```diff
@@ -24,13 +24,12 @@ class HNSWTieredIndexTest : public ::testing::Test {
         return tiered_index->getHNSWIndex();
     }
     TieredHNSWIndex<data_t, dist_t> *CreateTieredHNSWIndex(VecSimParams &hnsw_params,
-                                                           tieredIndexMock::IndexExtCtx *ctx,
+                                                           tieredIndexMock &mock_thread_pool,
```

Comment:
I think that there is a way to define a ctor to the test. We can just put the threadpool initialization there

Reply:
But it's not required for all tests...